pennylane/interfaces/jax_jit_tuple.py

```diff
@@ -203,6 +218,10 @@ def wrapper(p):
     def execute_wrapper_jvp(primals, tangents):
         # pylint: disable=unused-variable
         params = primals[0]
+        # Select the trainable params
+        # Non trainable params are 0
```

Comment:
I do not quite understand this comment, sorry :grimacing: What exactly is 0, and how does it relate to the following code line?

Reply:
Parameters here are also parameters that are jitted but not trainable, the non-trainable will contribute 0 for the JVP.